docs/ref/contrib/admin/index.txt

```diff
@@ -2355,6 +2355,17 @@ The ``InlineModelAdmin`` class adds or customizes:
     Specifies whether or not inline objects that can be changed in the
     admin have a link to the change form. Defaults to ``False``.
 
+.. method:: InlineModelAdmin.get_fieldsets(request, obj=None)
+
+    The ``get_fieldsets`` method is given the ``HttpRequest`` and the ``obj``
+    being edited (or ``None`` on an add form) and is expected to return a list
+    of two-tuples, in which each two-tuple represents a ``<fieldset>`` on the
+    admin form page, as described above in the :attr:`ModelAdmin.fieldsets` section.
```

Comment:
`InlineModelAdmin.get_fieldsets()` works exactly the same as `ModelAdmin.get_fieldsets()` i.e. `InlineModelAdmin` doesn't customize this class. I don't think that repeating the same description as for ``ModelAdmin.get_fieldsets()`` has any value. I would add `get_fieldsets()` to the list of shared methods (above) and an overall note at the end, e.g.
```
.. note::
    ``obj`` passed to ``InlineModelAdmin`` methods is the parent object being
    edited or ``None`` when adding a new parent.
```

Reply:
Yes indeed, I think that makes more sense. Sure I will change it